central/complianceoperator/v2/scanconfigurations/datastore/datastore_impl.go

```diff
@@ -85,17 +85,44 @@ func (ds *datastoreImpl) UpsertScanConfiguration(ctx context.Context, scanConfig
 }
 
 // DeleteScanConfiguration deletes the scan configuration specified by id
-func (ds *datastoreImpl) DeleteScanConfiguration(ctx context.Context, id string) error {
+func (ds *datastoreImpl) DeleteScanConfiguration(ctx context.Context, id string) (string, []string, error) {
+	clustersDeleted := []string{}
+	scanConfigName := ""
 	if ok, err := scanConfigurationsSAC.WriteAllowed(ctx); err != nil {
-		return err
+		return scanConfigName, clustersDeleted, err
```

Comment:
We tend to return empty values when we return an err.  In this instance, you are, but down below as you populate the scanConfigName and the clustersDeleted, you are not necessarily returning the empty values.  In the grand scheme of things it shouldn't matter but it may fit more with the typical ACS pattern to make all these that return error follow the pattern of explicitly returning "" and nil for the name and clusters.